sub.go

```diff
@@ -40,6 +40,47 @@ const (
 	SubscriptionPauseCollectionBehaviorVoid              SubscriptionPauseCollectionBehavior = "void"
 )
 
+// Preferred language of the Bancontact authorization page that the customer is redirected to.
+type SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage string
+
+// List of values that SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage can take
+const (
+	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageDe SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "de"
+	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageEn SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "en"
+	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageFr SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "fr"
+	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageNl SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "nl"
```

Comment:
I think these are acronyms and need to be properly gophercased
```suggestion
const (
	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageDE SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "de"
	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageEN SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "en"
	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageFR SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "fr"
	SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguageNL SubscriptionPaymentSettingsPaymentMethodOptionsBancontactPreferredLanguage = "nl"
```

Reply:
Although elsewhere, it looks like we've kind of punted on this problem and just have "preferred_language" as a string. I think that probably makes sense here, too.